hive/utils/rliable.py

```diff
@@ -0,0 +1,323 @@
+from rliable import library as rly
+from rliable import metrics
+from rliable import plot_utils
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+from scipy.stats.stats import find_repeats
+import matplotlib.patches as mpatches
+from matplotlib.ticker import MaxNLocator
+import argparse
+
+#Aggregate metrics
+IQM = lambda x: metrics.aggregate_iqm(x) # Interquartile Mean
+OG = lambda x: metrics.aggregate_optimality_gap(x, 1.0) # Optimality Gap
+MEAN = lambda x: metrics.aggregate_mean(x)
+MEDIAN = lambda x: metrics.aggregate_median(x)
+
+def score_normalization(score_, min_scores, max_scores):
+  norm_scores = (score_.copy() - float(min_scores))/(float(max_scores) - float(min_scores))
+  return norm_scores
+
+def save_fig(fig, name):
+  file_name = '{}.pdf'.format(name)
+  fig.savefig(file_name, format='pdf', bbox_inches='tight')
+
+def convert_to_matrix(score_dict):
+   keys = sorted(list(score_dict.keys()))
+   return np.stack([score_dict[k] for k in keys], axis=1)
+
+def load_and_read_rewards(env_names,path,min_env_score,max_env_score,Normalization):
+  score_dict = {}
+  for env_index,env_name in enumerate(env_names):
+    score_ = np.load(path+'/'+env_name+'/Rewards.npy')
```

Comment:
1. Use os.path.join
2. Where are you getting this file name from? You should be loading the rewards from the chomplogger files.